source/developer/mobile-developer-setup.rst

```diff
@@ -14,6 +14,82 @@ If you run into any issues getting your environment set up, check the Troublesho
 Development Environment Setup
 ---------------------------------
 
+We strongly recommend the use of a computer that supports macOS to develop and build the Mattermost mobile apps as macOS is required to build the iOS app.
```

Comment:
If macOS is a pre-requisite, propose changing to:

"A macOS computer is required to build the Mattermost iOS mobile app."

Or, is it possible to build the Android app on a non-macOS computer?  This was a little unclear to me...